conf.py

```diff
@@ -186,10 +187,12 @@
     # The ``CodeToRestSphinx`` extension creates a file named
     # ``sphinx-enki-info.txt``, which should be ignored by Sphinx.
     "sphinx-enki-info.txt",
+    # TODO: Notes here. I assume this is produced by the coverage run in the test suite? A link to that point in the code would be nice.
     "test/htmlcov",
-    "alembic/versions",
-    "**/.mypy_cache",
+    # TODO: Why ignore this?
```

Comment:
I think there is some philosophy that we need to talk through about what we include in the document and how we organize it so that a newcomer can easily find what they need to get started and be useful.  Right now the documents we produce are a bit like looking for a needle in a haystack for important stuff.  IMHO we should not bury big picture stuff in the details of an implementation.  And we shouldn't overwhelm the user by including too many things.

Reply:
I started a Wiki page to help us organize big-picture docs at https://github.com/bnmnetp/BookServer/wiki/Overall-documentation-structure.